gulpfile.js

```diff
@@ -69,14 +69,12 @@ gulp.task('depcheck', () => new Promise((resolve, reject) => {
     let invalidFiles = Object.keys(result.invalidFiles) || [];
     let invalidJsFiles = invalidFiles.filter((f) => f.endsWith('.js'));
     if (invalidJsFiles.length > 0) {
-      console.log('Invalid files:', result.invalidFiles);
-      reject(new Error('Invalid files'));
+      reject(new Error(`Invalid files: ${result.invalidFiles}`));
       return;
     }
 
     if (result.dependencies.length) {
-      console.log('Unused dependencies:', unused.dependencies);
```

Comment:
Gulp just silently failed here. It looks like Gulp is swallowing exceptions. I'll continue to dig...